float64 time
float32 depth
geometry_msgs/Vector3 linear_velocity # body x y z
geometry_msgs/Vector3 orientation # x y z
geometry_msgs/Vector3 linear_acceleration # body x y z
geometry_msgs/Vector3 angular_velocity # x y z 

